### PR TITLE
[new release] melange-recharts (4.0.3)

### PR DESCRIPTION
**Pull request by liubko**
Melange bindings for recharts

- Project page: <a href="https://github.com/ahrefs/melange-recharts/">https://github.com/ahrefs/melange-recharts/</a>

##### CHANGES:

- add `filterNull` to `Tooltip`
- support `melange.3.0.0` and `reason-react.0.14.0`

### CODE REVIEW

File: packages/melange-recharts/melange-recharts.4.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+homepage: "https://github.com/ahrefs/melange-recharts/"
+bug-reports: "https://github.com/ahrefs/melange-recharts/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml"
+  "melange" {>= "3.0.0"}
+  "reason" {>= "3.10.0"}
+  "reason-react" {>= "0.14.0"}
+  "reason-react-ppx" {>= "0.14.0"}
+  "ocaml-lsp-server" {with-test}
```
Comment (mseri) [inline packages/melange-recharts/melange-recharts.4.0.3/opam:21]:
```suggestion
  "ocaml-lsp-server" {with-dev-setup}
```
Is it really a test dependency or just a dev setup one?

Comment (jchavarri) [reply packages/melange-recharts/melange-recharts.4.0.3/opam:21]: I think we can remove this package from the deps directly. The reason why I'd rather not use `with-dev-setup` is that the current version of opam (2.1) doesn't support it, so there's no way to call `opam install --with-dev-setup` until 2.2 is out.

Comment (liubko) [reply packages/melange-recharts/melange-recharts.4.0.3/opam]: I removed `ocaml-lsp-server` dependency

**Comment by mseri** [comment]
Thanks

**Pull request merged**